docs/tutorial.py

```diff
@@ -52,9 +52,9 @@
 >>> await dev.update()
 >>> light.brightness
 50
->>> light.is_color
+>>> light.has_feature(light.set_hsv)
```

Comment:
I'd use `hsv` here, or would it even make more sense to use `Light.hsv` to make it explicit that it's a feature of the interface?

How I'm feeling about the `has_` on setters especially is that one should not really be using those that much–even when it works–but rather try to set the notion that it's about an interface feature in the first place, even when it happens to work with the "magic" involved in the background.

As we have rather few write-only features, it somehow makes more sense in my mind to have it this way. What do you think?

Reply:
That's updated now to use the getter name strings.